src/status_im/contexts/wallet/wallet_connect/responding_events.cljs

```diff
@@ -106,7 +106,11 @@
                  :method               method
                  :wallet-connect-event event
                  :event                :wallet-connect/on-sign-error})
-     {:fx [[:dispatch [:wallet-connect/dismiss-request-modal]]]})))
+     {:fx [[:dispatch [:wallet-connect/dismiss-request-modal]]
+           [:dispatch
+            [:toasts/upsert
```

Comment:
Do we have a generic `something-went-wrong` toast dispatch handler ?

Reply:
Nope, we don't.